ambry-account/src/test/java/com/github/ambry/account/CompositeAccountServiceTest.java

```diff
@@ -0,0 +1,264 @@
+/*
+ * Copyright 2020 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.account;
+
+import com.codahale.metrics.MetricRegistry;
+import com.github.ambry.config.CompositeAccountServiceConfig;
+import com.github.ambry.config.VerifiableProperties;
+import com.github.ambry.utils.TestUtils;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Properties;
+import java.util.Set;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.*;
+
+
+public class CompositeAccountServiceTest {
+
+  AccountService primaryAccountService = mock(HelixAccountService.class);
+  AccountService secondaryAccountService = mock(MySqlAccountService.class);
+  AccountServiceMetrics metrics = new AccountServiceMetrics(new MetricRegistry());
+  Properties props = new Properties();
+  AccountService compositeAccountService;
+
+  public CompositeAccountServiceTest() {
+    compositeAccountService = new CompositeAccountService(primaryAccountService, secondaryAccountService, metrics,
+        new CompositeAccountServiceConfig(new VerifiableProperties(props)));
+  }
+
+  /**
+   * Test composite getAccountById() with both sources returning the same result.
+   */
+  @Test
+  public void testGetAccountByIdBothSuccess() {
+    Account testAccount = getTestAccount();
```

Comment:
`getTestAccount()` is called in several places and the method instantiates same account/container several times. Why not have a global `testAccount`?